### PR TITLE
one call API for openweathermap

### PR DESCRIPTION
**Pull request by cinhcet**
# Types of changes
openweathermap has a new api https://openweathermap.org/api/one-call-api which allows to fetch hourly forecasts for 48 hours and daily forecasts for 7 days as well as the current weather in one call. The response is similar to dark sky (not equal, but on the openweathermap site there is a kind of upgrade guide)

What types of changes does your code introduce?
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Proposed changes
New option to use the new API.

## Checklist
- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: openweathermap/locales/en-US/weather.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
             "location": "Location",
             "option_current": "Current weather for",
             "option_5dayforecast": "5 day forecast for",
+            "option_onecall": "combined current wheather/forecast for",
```
Comment (Paul-Reed) [inline openweathermap/locales/en-US/weather.json:9]: TYPO - weather not wheather

Comment (cinhcet) [reply openweathermap/locales/en-US/weather.json]: ups :-) fixed

**Comment by dceejay** [comment]
Sorry - as you noted - got lost in the general business of 1.1.
Do feel free to ding us on the issue/pr when we fail to notice...

**Comment by cinhcet** [comment]
Just a tiny thing left, I have used httpS instead of http, I wonder if it should be http like in the other calls?

Sorry for not bringing this up earlier. I can make a new PR if you want

**Pull request merged**